lib/store/src/autoTitle.ts

```diff
@@ -36,7 +35,9 @@ export const autoTitleFromSpecifier = (fileName: string, entry: NormalizedStorie
 
   if (importPathMatcher.exec(normalizedFileName)) {
     const suffix = normalizedFileName.replace(directory, '');
-    const titleAndSuffix = slash(path.join(titlePrefix, suffix));
+    const slashes = new RegExp('/{1,}', 'g');
+    // Do not use path to join the parts, since this runs in the browser
+    const titleAndSuffix = slash([titlePrefix, suffix].join('/').replace(slashes, '/'));
```

Comment:
Can you extract this into a standalone function `pathJoin` just to make it a little more readable?